.github/workflows/end-to-end-test.yml

```diff
@@ -1,6 +1,10 @@
 name: Runs end to end tests
 on:
-  workflow_call:
+  # workflow_call:
+  # workflow_dispatch:
+  pull_request:
+    branches:
+      - main
```

Comment:
**comment**: I don't think that these tests are ready to run against every pull request to main. Let's put this whole section back to just `workflow_call:` as it is on `main`.